analysis/shamanenhancement/src/modules/talents/HotHand.tsx

```diff
@@ -21,19 +23,63 @@ const HOT_HAND = {
  *
  */
 class HotHand extends Analyzer {
+  static dependencies = {
+    spellUsable: SpellUsable,
+    haste: Haste,
+  };
+  protected spellUsable!: SpellUsable;
+  protected haste!: Haste;
+
   protected buffedLavaLashDamage: number = 0;
 
   constructor(options: Options) {
     super(options);
 
     this.active = this.selectedCombatant.hasTalent(SPELLS.HOT_HAND_TALENT.id);
 
+    this.addEventListener(
+      Events.applybuff.by(SELECTED_PLAYER).spell(SPELLS.HOT_HAND_BUFF),
+      this.reduceLavaLushCDOnBuff,
```

Comment:
these should be `LavaLash` not `LavaLush`

Reply:
Oh my! Thanks for noting this! Changes have been mande.